### PR TITLE
Fix "stupid" fakematch

### PR DESCRIPTION
**Pull request by PokeCodec**
Sometimes the answer is the simplest one.

### CODE REVIEW

File: src/link_rfu_2.c [hunk 1]
```diff
@@ -1555,28 +1552,23 @@ static bool8 CheckForLeavingGroupMembers(void)
...
     for (i = 0; i < RFU_CHILD_MAX; i++)
     {
-        if (Rfu.partnerSendStatuses[i] < RFU_STATUS_JOIN_GROUP_OK 
-         || Rfu.partnerSendStatuses[i] > RFU_STATUS_JOIN_GROUP_NO)
+        if (Rfu.partnerSendStatuses[i] == RFU_STATUS_JOIN_GROUP_OK || Rfu.partnerSendStatuses[i] == RFU_STATUS_JOIN_GROUP_NO)
+            continue;
+        if (gRfuSlotStatusNI[i]->recv.state == SLOT_STATE_RECV_SUCCESS || gRfuSlotStatusNI[i]->recv.state == SLOT_STATE_RECV_SUCCESS_AND_SENDSIDE_UNKNOWN)
         {
-            if (gRfuSlotStatusNI[i]->recv.state == SLOT_STATE_RECV_SUCCESS 
-             || gRfuSlotStatusNI[i]->recv.state == SLOT_STATE_RECV_SUCCESS_AND_SENDSIDE_UNKNOWN)
```
Comment (GriffinRichards) [inline src/link_rfu_2.c:1562]: Restore the alignment on these

**Comment by PokeCodec** [comment]
Fixed!

File: src/link_rfu_2.c [hunk 2]
```diff
@@ -1555,28 +1548,25 @@ static bool8 CheckForLeavingGroupMembers(void)
...
-        if (Rfu.partnerSendStatuses[i] < RFU_STATUS_JOIN_GROUP_OK 
-         || Rfu.partnerSendStatuses[i] > RFU_STATUS_JOIN_GROUP_NO)
+        if (Rfu.partnerSendStatuses[i] == RFU_STATUS_JOIN_GROUP_OK
+            || Rfu.partnerSendStatuses[i] == RFU_STATUS_JOIN_GROUP_NO)
+            continue;
+        if (gRfuSlotStatusNI[i]->recv.state == SLOT_STATE_RECV_SUCCESS
+            || gRfuSlotStatusNI[i]->recv.state == SLOT_STATE_RECV_SUCCESS_AND_SENDSIDE_UNKNOWN)
         {
-            if (gRfuSlotStatusNI[i]->recv.state == SLOT_STATE_RECV_SUCCESS 
-             || gRfuSlotStatusNI[i]->recv.state == SLOT_STATE_RECV_SUCCESS_AND_SENDSIDE_UNKNOWN)
```
Comment (GriffinRichards) [inline src/link_rfu_2.c:1562]: They're still not aligned, see how they were originally

File: src/battle_main.c [hunk 3]
```diff
@@ -4851,40 +4851,34 @@ static void SetActionsAndBattlersTurnOrder(void)
...
+                u8 battler1 = gBattlerByTurnOrder[i];
+                u8 battler2 = gBattlerByTurnOrder[j];
+                if (gActionsByTurnOrder[i] != B_ACTION_USE_ITEM && gActionsByTurnOrder[j] != B_ACTION_USE_ITEM && gActionsByTurnOrder[i] != B_ACTION_SWITCH && gActionsByTurnOrder[j] != B_ACTION_SWITCH)
                 {
-                    u8 battler1 = gBattlerByTurnOrder[i];
-                    u8 battler2 = gBattlerByTurnOrder[j];
-                    if (gActionsByTurnOrder[i] != B_ACTION_USE_ITEM
-                        && gActionsByTurnOrder[j] != B_ACTION_USE_ITEM
-                        && gActionsByTurnOrder[i] != B_ACTION_SWITCH
-                        && gActionsByTurnOrder[j] != B_ACTION_SWITCH)
```
Comment (GriffinRichards) [inline src/battle_main.c:4883]: Same here now

**Comment by PokeCodec** [comment]
Finished

**Pull request merged**